changelogs/fragments/1804_enhance_error_message_when_failing_import.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - module_utils/import_handler - Change function call to ensure get full traceback when failing import.
```

Since this will be a customer facing entry in the release notes, I think we should not mention "function" or "full traceback" since those have little to no meaning in the user space.

Even though is not a bug we can use past tense to explain past behavior.

```suggestion
  - module_utils/import_handler - When importing a non supported ZOAU version like 1.2.x the module would throw a non user friendly error message.
    Error message is now explicit about ZOAU not being properly configured for Ansible.
```

---

Agreed